src/sagas/backend.ts

```diff
@@ -275,6 +276,80 @@ function* backendSaga(): SagaIterator {
       handleResponseError(resp);
     }
   });
+
+  yield takeEvery(actionTypes.FETCH_NOTIFICATIONS, function*(action) {
+    const tokens = yield select((state: IState) => ({
+      accessToken: state.session.accessToken,
+      refreshToken: state.session.refreshToken
+    }));
+
+    if (!tokens.accessToken) {
+      return;
+    }
+
+    const resp: Response | null = yield request('notification', 'GET', {
```

Comment:
Why not follow convention and have a separate function to call the request? (similar to our other sagas)

Reply:
Refactored accordingly